cfgov/jinja2/v1/offices/_single.html

```diff
@@ -147,17 +147,20 @@ <h3 class="h2 qa-resource-title">
         {{ office.content | safe }}
     </section>
     {% endif %}
-    <aside>
+    <aside class="block
```

Comment:
Discussed in chat and I'm pulling it out since some of it will carry over to the new project and be used in multiple places.